DuckDuckGo/AuthenticationViewController.swift

```diff
@@ -58,18 +58,37 @@ class AuthenticationViewController: UIViewController {
             onCouldNotAuthenticate()
         }
     }
+    
+    class Authenticator {
```

Comment:
We shouldn't nest a new `Authenticator` class in here - instead, the [original class](https://github.com/duckduckgo/iOS/blob/develop/DuckDuckGo/Authenticator.swift) should be modified.

Furthermore, the original class [already has an `async` variant](https://github.com/duckduckgo/iOS/blob/178157a7896bf976076a89a3e9a7fbc751edfa5b/DuckDuckGo/Authenticator.swift#L46) of the authenticate function, so this class isn't necessary.